governance/doc.go

```diff
@@ -14,5 +14,51 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
 
-// Package governance contains functions and variables used for voting and reflecting vote results in Klaytn.
+/*
+Package governance contains functions and variables used for voting and reflecting vote results in Klaytn.
+In Klaytn, various settings such as the amount of KLAY minted as a block reward can be changed by using governance vote.
+These votes can be casted by nodes of Governance Council and detailed introduction can be found at <https://docs.klaytn.com/klaytn/governance>
+
+How to cast a vote
+
+To cast a vote, a node have to be a member of the Governance Council.
+If the governance mode is "single", only one designated node (the governing node) can vote.
+In the console of the node, "governance.vote(key, value)" API can be used to cast a vote.
+
+Keys for the voting API
+
+Following keys can be handled as of 7/20/2019.
+- "governance.governancemode"   : To change the governance mode
+- "governance.governingnode"    : To change the governing node if the governance mode is "single"
+- "governance.unitprice"        : To change the unitprice of Klaytn. (Unit price is same as gasprice in Ethereum)
```

Comment:
let's remove a period at the end of a sentence like other lines.

Reply:
Thanks! It is updated.